src/main/resources/application.yaml

```diff
@@ -157,6 +157,7 @@ reconciliation:
   report:
       mail-from: ${SMTP_USERNAME}
       mail-recipients: ${RECONCILIATION_REPORT_RECIPIENTS}
+      skip-empty: ${RECONCILIATION_REPORT_SKIP_EMPTY:true}
```

Comment:
do we need configuration?

Reply:
Even if business does not need config right now I assume having this configurable will not be bad.

Reply:
I think we do not need. there is a config for email sending. without file there is no point to send email